blog/2023-11-13-introducing-stately-sky/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Introducing Stately Sky'
+description: Announcing Stately Sky, the new way to run your state machines in the cloud.
+authors: [gavin]
+tags: [new, sky, serverless, cloud, run, machines, state, stately]
+date: 2023-11-13
+slug: 2023-11-13-introducing-stately-sky
+image: /blog/2023-11-13-introducing-stately-sky.png
+---
+
+The Stately team is very excited to announce a new feature we’ve been working on for quite some time! Join us in welcoming Stately Sky to the Studio. Lovingly built with [PartyKit](https://www.partykit.io/), Sky is our new serverless platform for running workflows within the Studio. With Sky, users may now run their statecharts as live machines in minutes, complete with XState v5 actors and multiplayer support.
+
+<!--truncate-->
+
+## Why Sky: a seamless developer experience without the lock-in(Working title)
```

Comment:
```suggestion
## Why Sky: a seamless developer experience for live machines and multiplayer in the cloud
```

Is the lack of lock-in our key feature? This suggestion (just a suggestion!) takes inspiration from your intro.

Reply:
This is a good point. Perhaps I should have even mentioned this in the intro. I think ( and correct me if I'm wrong) that while it isn't our "key" feature, it is one that many firms care about. Building your business on AWS is all fine until they jack up prices, and you can't easily port it elsewhere, for example. Since that affects the bottom line, I thought it necessary to pop it in the header here as a value add, but would love your take.